iron-selectable.html

```diff
@@ -132,6 +132,8 @@
     attached: function() {
       this._observer = this._observeItems(this);
       this._contentObserver = this._observeContent(this);
+      this._removeListener(this.activateEvent);
```

Comment:
To be more clear, I think that if we are adding this listener in two places, we should change it to only add it in one, rather then thrash the listener needlessly on `attached`.